### PR TITLE
feat(fa): Income form added to application

### PR DESCRIPTION
**Pull request by MargretFinnboga**
# Income form added

https://app.asana.com/0/1201600280110882/1201624216643410

## What

 - Income form added to application
 - Added approve option message for usability
 - Fixing message in about form
 - Fixing error messages 

## Why

- we need income info

## Screenshots / Gifs
Screenshot 2022-01-12 at 13 14 39

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/financial-aid/src/fields/IncomeForm/IncomeForm.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        <BulletList type={'ul'} space={2}>
+          <Bullet>
+            {formatMessage(incomeForm.examplesOfIncome.fromEmployer)}
+          </Bullet>
+          <Bullet>
+            {formatMessage(incomeForm.examplesOfIncome.fromDirectorateOfLabor)}
+          </Bullet>
+          <Bullet>
+            {formatMessage(incomeForm.examplesOfIncome.fromInsurance)}
+          </Bullet>
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/fields/IncomeForm/IncomeForm.tsx:49]: Can this be one translation string?

File: libs/application/templates/financial-aid/src/fields/IncomeForm/IncomeForm.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        <BulletList type={'ul'} space={2}>
+          <Bullet>
+            {formatMessage(incomeForm.examplesOfIncome.fromMaternityLeaveFund)}
+          </Bullet>
+          <Bullet>
+            {formatMessage(incomeForm.examplesOfIncome.fromHealthInsurance)}
+          </Bullet>
+          <Bullet>
+            {formatMessage(incomeForm.examplesOfIncome.fromPensionFund)}
+          </Bullet>
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/fields/IncomeForm/IncomeForm.tsx:60]: And this?

File: libs/application/templates/financial-aid/src/fields/IncomeForm/IncomeForm.tsx (hunk 1 above)
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/fields/IncomeForm/IncomeForm.tsx:49]: Can this be one translation string?

File: libs/application/templates/financial-aid/src/fields/IncomeForm/IncomeForm.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        </BulletList>
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/fields/IncomeForm/IncomeForm.tsx:61]: And this?

File: libs/application/templates/financial-aid/src/forms/Application.ts [hunk 4]
```diff
@@ -67,5 +67,28 @@ export const Application: Form = buildForm({
...
+    buildSection({
+      id: '',
+      title: '',
+      children: [
+        buildCustomField({
+          id: '',
+          title: '',
+          component: '',
+        }),
+      ],
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/forms/Application.ts:91]: Looks like some merge conflict gone wrong here

File: libs/application/templates/financial-aid/src/lib/messages/application.ts [hunk 5]
```diff
@@ -21,6 +21,19 @@ export const section = defineMessages({
   },
 })
 
+export const approveOptions = defineMessages({
+  no: {
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/lib/messages/application.ts:25]: Nice 👏

**Review by Rikkiabb — CHANGES_REQUESTED** [review]

File: libs/application/templates/financial-aid/src/forms/Application.ts (hunk 4 above)
Comment (MargretFinnboga) [reply libs/application/templates/financial-aid/src/forms/Application.ts:91]: Nope I added this also to get next buttons, and the comment got to follow along 🙃

Comment (Rikkiabb) [reply libs/application/templates/financial-aid/src/forms/Application.ts:91]: Ahh I see and we will remove it at the end of the flow then?

File: libs/application/templates/financial-aid/src/fields/DescriptionText/DescriptionText.tsx [hunk 6]
```diff
@@ -12,11 +12,7 @@ import {
 import * as styles from './DescriptionText.css'
 
 const BulletListBox = ({ children }: { children: ReactNode }) => {
-  return (
-    <Box marginBottom={3}>
-      <BulletList space={2}>{children}</BulletList>
-    </Box>
-  )
+  return <BulletList space={2}>{children}</BulletList>
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/fields/DescriptionText/DescriptionText.tsx:15]: Should rename the const as well, this is not a bulletlistbox any more :)

**Review by Rikkiabb — APPROVED** [review]

**Pull request merged**